main.js

```diff
@@ -0,0 +1,161 @@
+document.querySelector('.search').addEventListener('click', () => {
+  const location = document.querySelector('#search-query').value;
+
+  document.querySelector('#search-query').value = '';
+
+  fetchCurrentConditions(location);
+  fetchFiveDayForecast(location);
+
+  
+});
+
+const fetchCurrentConditions = (location) => {
+  const url = `https://api.openweathermap.org/data/2.5/weather?q=${location}&units=imperial&appid=79a2d684772528c31363d342e6db9789`;
+
+  fetch(url, {
+    method: 'GET',
+    dataType: 'json'
+  })
+    .then(data => data.json())
+    .then(data => addCurrentConditions(data));
+}
+
+const fetchFiveDayForecast = (location) => {
+  const urlForecast = `https://api.openweathermap.org/data/2.5/forecast?q=${location}&units=imperial&cnt=40&appid=79a2d684772528c31363d342e6db9789`; 
+
+  fetch(urlForecast, {
+    method: 'GET',
+    dataType: 'json'
+  })
+    .then(dataForecast => dataForecast.json())
+    .then(dataForecast => addFiveDayForecast(dataForecast));
+}
+
+
+const addCurrentConditions = (data) => {
+  const currentConditions = [];
+
+  currentConditions.push({
+    temp: data.main.temp,
+    name: data.name,
+    weather: data.weather[0].main,
+    weatherIcon: data.weather[0].icon
+    
+  })
+  renderCurrentConditions(currentConditions);
+}
+
+const addFiveDayForecast = (dataForecast) => {
+  const fiveDayForecast = [];
+
+  fiveDayForecast.push({
+    weather1: dataForecast.list[5].weather[0].main,
```

Comment:
while hardcoding indexes might work for this, think about what if the api changes and the data is at different indexes - how could this be changed